src/sql-server-util/src/cdc.rs

```diff
@@ -310,60 +356,114 @@ pub enum CdcError {
 /// function.
 ///
 /// [`sys.fn_cdc_increment_lsn`](https://learn.microsoft.com/en-us/sql/relational-databases/system-functions/sys-fn-cdc-increment-lsn-transact-sql?view=sql-server-ver16)
```

Comment:
This doc comment block needs some updates since LSNs are not treated as opaque blobs anymore and we increment them without calling this function

Reply:
Updated!